Missing property documentation

KeyboardRage opened the pull request:
The interaction received from Discord also includes an `application_id` field.

Snazzah reviewed (changes requested):
1. Tests need to be updated to account for the new property
2. This should also be added to `PingRequestData`, `DMMessageComponentRequestData`, `GuildMessageComponentRequestData`, `DMCommandAutocompleteRequestData` and `GuildCommandAutocompleteRequestData`

KeyboardRage commented:
> 1. Tests need to be updated to account for the new property
> 2. This should also be added to `PingRequestData`, `DMMessageComponentRequestData`, `GuildMessageComponentRequestData`, `DMCommandAutocompleteRequestData` and `GuildCommandAutocompleteRequestData`

Added to tests, not sure if I got it all though.
Also added the property to the mentioned types.
